packages/api-fetch/src/wp-module.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Internal dependencies
+ */
+import { createNonceMiddleware } from './middlewares/nonce';
+import { createRootURLMiddleware } from './middlewares/root-url';
+import { createPreloadingMiddleware } from './middlewares/preloading';
+import { fetchAllMiddleware } from './middlewares/fetch-all-middleware';
+import { mediaUploadMiddleware } from './middlewares/media-upload';
+import { createThemePreviewMiddleware } from './middlewares/theme-preview';
+import { apiFetch, setFetchHandler } from './core';
+import { registerMiddleware } from './middlewares/singleton';
+
+if ( typeof document !== 'undefined' ) {
+	const el = document.getElementById(
+		'wp-scriptmodule-data_@wordpress/api-fetch'
```

Comment:
This should be updated with the correct ID:

https://github.com/WordPress/gutenberg/blob/5738e9b0529eae4678539e28d83747258824df5d/lib/experimental/script-modules.php#L284